src/buffer/out/textBuffer.cpp

```diff
@@ -542,14 +542,22 @@ bool TextBuffer::NewlineCursor()
 // - <none>

```

Comment:
Arguments comment not updated.

Reply:
Sorry I totally overlooked that. Added in commit 171a4f9dddfef7608fba7f9302050d78e4de66cc